widgets/Zuora-batchsource.json

```diff
@@ -0,0 +1,170 @@
+                "label": "OAuth2"
+              }
+            ]
+          }
+        },
+        {
+          "name": "authUserName",
+          "label": "Username",
+          "widget-type": "textbox"
+        },
+        {
+          "name": "authPassword",
+          "label": "Password",
+          "widget-type": "password"
+        },
+        {
+          "name": "authClientId",
+          "label": "Client ID",
+          "widget-type": "securekey-text",
+          "widget-attributes": {
+            "placeholder": "OAuth2 Client ID"
+          }
+        },
+        {
+          "name": "authClientSecret",
+          "label": "Client Secret",
+          "widget-type": "securekey-text",
+          "widget-attributes": {
+            "placeholder": "OAuth2 Client Secret"
+          }
+        },
+        {
+          "name": "apiEndpoint",
+          "label": "REST API Endpoint",
+          "widget-type": "select",
+          "widget-attributes": {
+            "values": [
+              "https://rest.zuora.com",
+              "https://rest.eu.zuora.com",
+              "https://rest.apisandbox.zuora.com",
+              "https://rest.sandbox.eu.zuora.com",
+              "https://rest.pt1.zuora.com"
+            ],
+            "default": "https://rest.zuora.com"
+          }
+        },
+        {
+          "name": "baseObjects",
+          "label": "Base Objects",
+          "widget-type": "multi-select",
+          "widget-attributes": {
+            "options": [
+              {
+                "id": "Product",
+                "label": "Product"
+              },
+              {
+                "id": "Refund",
+                "label": "Refund"
+              },
+              {
+                "id": "PaymentRun",
+                "label": "PaymentRun"
+              },
+              {
+                "id": "Orders",
+                "label": "Orders"
+              },
+              {
+                "id": "Entities",
+                "label": "Entities"
+              },
+              {
+                "id": "DebitMemos",
+                "label": "DebitMemos"
+              },
+              {
+                "id": "CreditMemos",
+                "label": "CreditMemos"
+              },
+              {
+                "id": "BillingDocuments",
+                "label": "BillingDocuments"
+              }
+            ],
+            "delimiter": ","
+          }
+        },
+        {
+          "name": "advObjects",
+          "label": "Advanced Objects",
+          "widget-type": "multi-select",
+          "widget-attributes": {
+            "options": [
+            ],
+            "delimiter": ","
+          }
+        }
+      ]
+    }
+  ],
+  "outputs": [
+    {
+      "widget-type": "non-editable-schema-editor",
+      "schema": {
+      }
+    }
+  ],
+  "filters": [
+    {
+      "name": "AuthTypeBasicFilter",
+      "condition": {
+        "expression": "authType == 'basic'"
```

Comment:
According to Albert we should use named operators everywhere it's possible.
https://github.com/data-integrations/http/blob/develop/widgets/HTTP-batchsource.json#L523